src/components/TextField/TextField.tsx

```diff
@@ -47,14 +64,28 @@ export const TextField: React.FC<Props> = ({
           })}
           placeholder={placeholder}
           value={value}
-          onChange={event => onChange(event.target.value)}
-          onBlur={() => setTouched(true)}
+          onChange={event => {
```

Comment:
```suggestion
          onChange={your_function_name}
```
Consider moving this logic from jsx